### PR TITLE
User entity and State enum added. TODOs added for Muhabbat and Gulmira

### PR DESCRIPTION
**Pull request by RumCake01**
my first commit :money_mouth_face:

### CODE REVIEW

**Comment by VitalySlabada** [comment]
@RumCake01  can you please create data in data sql to test your changes.

**Review by BahaMaksudov — COMMENTED** [review]
@RumCake01 - Everything looks good, app runs successfully. As commented by Vitaly please add data in data.sql to be loaded in users table. Thank you!

File: src/main/java/com/example/accountingapp/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+@Where(clause= "is_deleted=false")
+public class User extends BaseEntity {
+
+    private String firstName;
+    private String LastName;
```
Comment (MehmetKaldirim) [inline src/main/java/com/example/accountingapp/entity/User.java]: if u change LastName to lastName will be great ...actually they are not camel case in ER...thank u

**Review by MehmetKaldirim — APPROVED** [review]
Great job.

File: src/main/java/com/example/accountingapp/entity/User.java (hunk 1 above)
Comment (RumCake01) [reply src/main/java/com/example/accountingapp/entity/User.java]: oh, yes. Thank you @MehmetKaldirim .  
I actually updated it, but then lost all my changes after I pulled updates from master, and then i did not notice this again. Will update today and push again.

**Review by BahaMaksudov — APPROVED** [review]
Looks nice! Apps runs and data loads successfully.

**Comment by RumCake01** [comment]
@BahaMaksudov thank you. WIll make a tiny change on the LastName field to change it to the lower case andp push it again when I have a second.

**Comment by mgulmira** [comment]
I push my changes to master, so please update your branch with most recent and try to uncomment todo in Company entity. It should work.

**Review by VitalySlabada — APPROVED** [review]

**Comment by VitalySlabada** [comment]
Whatever issue there was with product is fixed, once my PR is merged
no need to comment it

**Comment by mgulmira** [comment]
Should State be an entity? It has table in ERD and has relations with Company entity. Clarified on chat. will be enum. Apologies for the confusion

**Review by AbbosB12 — APPROVED** [review]

**Review by VitalySlabada — APPROVED** [review]
Looks good but could not test, please test before merging.

**Comment by VitalySlabada** [comment]
merged on other branch

**Pull request closed without merging**